pkg/controllers/machinesetsync/machineset_sync_controller.go

```diff
@@ -58,12 +64,13 @@ type MachineSetSyncReconciler struct {
 	MAPINamespace string
 }
 
-// SetupWithManager sets the CoreClusterReconciler controller up with the given manager.
+// SetupWithManager sets up the controller with the Manager.
 func (r *MachineSetSyncReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	infraMachineTemplate, err := getInfraMachineTemplateFromProvider(r.Platform)
 	if err != nil {
-		return fmt.Errorf("failed to get InfraMachine from Provider: %w", err)
+		return fmt.Errorf("failed to get InfraMachineTemplate from Provider: %w", err)
 	}
+
 	// Allow the namespaces to be set externally for test purposes, when not set,
 	// default to the production namespaces.
 	if r.CAPINamespace == "" {
```

Comment:
We could use `DefaultManagedNamespace` and `DefaultMAPIManagedNamespace` consts for these